src/ducks/settings/DelayedDebitAlertRules.jsx

```diff
@@ -73,28 +73,20 @@ const getModalProps = ({ initialDoc, t }) => ({
   initialDoc
 })
 
-class DelayedDebitCard extends React.Component {
-  static propTypes = {
-    // TODO replace `PropTypes.object` with a shape coming from cozy-doctypes
-    accounts: PropTypes.arrayOf(PropTypes.object).isRequired,
-    enabled: PropTypes.bool.isRequired,
-    onToggle: PropTypes.func.isRequired,
-    t: PropTypes.func.isRequired
-  }
-
-  getCreditCardDefaultValue() {
-    const { accounts } = this.props
-    const selectedAccount = (accounts.data || []).find(isCreditCardAccount)
-    return selectedAccount ? selectedAccount : null
-  }
+const getCreditCardDefaultValue = props => {
+  const { accounts } = props
+  const selectedAccount = (accounts.data || []).find(isCreditCardAccount)
+  return selectedAccount ? selectedAccount : null
+}
 
-  getCheckingsDefaultValue() {
-    const selectedCreditCard = this.getCreditCardDefaultValue()
-    return selectedCreditCard && selectedCreditCard.checkingsAccount
-      ? selectedCreditCard.checkingsAccount.data
-      : null
-  }
+const getCheckingsDefaultValue = props => {
+  const selectedCreditCard = getCreditCardDefaultValue(props)
+  return selectedCreditCard && selectedCreditCard.checkingsAccount
+    ? selectedCreditCard.checkingsAccount.data
+    : null
+}
 
+class DelayedDebitCard extends React.Component {
```

Comment:
Nit: converting it to a functional component (so we tend to use hooks in it in the future) must be kinda free now

Reply:
Not used anymore ;)